lib/parallel_tests/tasks.rb

```diff
@@ -56,10 +56,10 @@ def check_for_pending_migrations
         end
       end
 
-      # parallel:spec[:count, :pattern, :options]
+      # parallel:spec[:count, :pattern, :options, :serialize_stdout]
```

Comment:
I thought it could be a good idea because I found the rake task to be useful but not as much power as the executable but I recognize that serialize_stdout is not as much important like the others.

Maybe the 4th parameter could be a more generic options parameter which is passed directly to the executable?
If so the current options could be renamed :test_options.

What do you think about it?